resources/user-agents/browsers/misc/alienforce.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
For all of these new browsers I didn't add any versions to keep the pattern count low. Could be added later if necessary.